TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Knative Trademark committee"
+linkTitle: "Trademark committee"
+weight: 50
+type: "docs"
+aliases:
+  - /contributing/trademark-committee/
+---
+
+The Knative Trademark Committee (KTC)
+[advises the trademark owner on the usage and enforcement of the Knative mark](./GOVERNANCE.md).
+It is a separate body from the [Steering Committee](./STEERING-COMMITTEE.md) to
+better represent the needs of vendors and other companies which market products
+in conjuction with the Knative mark.
```

Comment:
Maybe it's sufficient to just say 'It is a separate body from the Steering Committee.' or even not justify its existence at all.

Reply:
I included the context in hopes that it would avoid needing to repeat all thees discussions in 2 years when all of us have gone on to bigger things. (Or in hopes that documenting the intent would help with interpretation of the spirit.)